fe/fe-core/src/main/java/org/apache/doris/analysis/CreateJobStmt.java

```diff
@@ -149,6 +150,8 @@ public void analyze(Analyzer analyzer) throws UserException {
         if (null != startsTimeStamp) {
             if (startsTimeStamp.equalsIgnoreCase(CURRENT_TIMESTAMP_STRING)) {
                 jobExecutionConfiguration.setImmediate(true);
+                //To avoid immediate re-scheduling, set the start time of the timer 100ms before the current time.
+                timerDefinition.setStartTimeMs(System.currentTimeMillis() - 100L);
```

Comment:
what will happen if not set startTimemMs?

Reply:
The syntax for Create Job doesn't immediately schedule the job. For a one-time schedule, setting the start time to CURRENT_TIMESTAMP achieves immediate scheduling. Therefore, we still need to set the startTime parameter; otherwise, the job query will lack a start time.